frontend/src/components/TicketReservationForm/index.jsx

```diff
@@ -0,0 +1,207 @@
+import React, { useState } from "react";
+import { useTicketReservationForm } from "./validation";
+import { ticketReservationAPI } from "./api";
+import { gender, specialNeeds, mealSpecification } from "./validation";
+import { countries, countryCodes } from "../signup/SignUpForm/validation";
+import Button from "../shared/Button";
+import Input from "../shared/Input";
+import "./style.css";
+const TicketForm = ({ numberOfTickets }) => {
+  const [currentTicket, setCurrentTicket] = useState(1);
+  const [tickets, setTickets] = useState([]);
+
+  const {
+    values,
+    errors,
+    touched,
+    isSubmitting,
+    handleChange,
+    handleBlur,
+    resetForm,
+    handleSubmit,
+  } = useTicketReservationForm((submittedValues) => {
+    setTickets((prevTickets) => [...prevTickets, submittedValues]);
+    if (currentTicket < numberOfTickets) {
+      setCurrentTicket(currentTicket + 1);
+      resetForm();
```

Comment:
the useTicketReservationForm hook resets the form after each ticket but there is no way to go back and edit a previous ticket.
Suggestion:
store ticket details in a state array and allow users to navigate back and forth between tickets for editing before submission.